doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1079 @@
+
+:ref:`gs_app_tcm`, used to install the |NCS| automatically from `nRF Connect for Desktop`_, is available for Windows, Linux, and macOS.
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.1.0_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.1.0_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.1.0`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_210_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Application development
+=======================
+
+* Added:
+
+  * Information about :ref:`gs_supported_OS` to the :ref:`gs_recommended_versions` page.
+  * Information about :ref:`app_build_output_files` on the :ref:`app_build_system` page.
+  * Information about :ref:`gs_debugging` on the :ref:`gs_testing` page.
+    Also added links to this section in different areas of documentation.
+  * Option to configure IEEE 802.15.4 ACK frame timeout at build time using :kconfig:option:`CONFIG_NRF_802154_ACK_TIMEOUT_CUSTOM_US`.
+
+* Updated :ref:`app_memory` page with sections about Gazell and NFC.
+
+RF Front-End Modules
+--------------------
+
+* Fixed a build error that occurred when building an application for nRF53 SoCs with Simple GPIO Front-End Module support enabled.
+* Added the :kconfig:option:`CONFIG_MPSL_FEM_ONLY` Kconfig option that allows the :ref:`nrfxlib:mpsl_fem` API to be used without other MPSL features.
+  The :ref:`MPSL library <nrfxlib:mpsl>` is linked into the build without initialization.
+  You cannot use other MPSL features when this option is enabled.
+* Added the possibility to add custom models which split the requested TX power between the power on SoC output and the FEM gain in a way desired by the user.
+
+Wi-Fi
+-----
+
+* Added support for the nRF7002 DK that supports Wi-Fi SoC.
+  For more information, see :ref:`ug_nrf70`.
+
+See `Wi-Fi samples`_ for details about how to use Wi-Fi in your application.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Bluetooth LE
+------------
+
```

Comment:
Please add "LTE dual-chip coex support on nRF52 is now ready for production"

Reply:
Updated.